tools/depcheck/glide/util.go

```diff
@@ -0,0 +1,43 @@
+package glide
+
+import (
+	"fmt"
+)
+
+// MissingImports receives a LockFile and a YamlFile and returns a slice of
+// imports not contained in the YamlFile. Dependencies with a non-empty "repo" field
+// are skipped, as they are assumed to contain forked dependencies.
+func MissingImports(lockfile *LockFile, yamlfile *YamlFile) (YamlFileImportList, error, []string) {
+	if lockfile == nil || yamlfile == nil {
+		return nil, fmt.Errorf("both a lockfile and a yamlfile are required"), []string{}
+	}
+
+	warnings := []string{}
+	newImports := []*YamlFileImport{}
+
+	for _, lockDep := range lockfile.Imports {
+		if len(lockDep.Repo) > 0 {
+			warnings = append(warnings, fmt.Sprintf("info: skipping package with \"repo\" field: %s\n", lockDep.Name))
```

Comment:
I'll take this in a tool, but generally I'd be looking to see the entire `[]lockDep` for skipped items.